.travis.yml

```diff
@@ -1,8 +1,15 @@
 sudo: false
 
+language: node_js
```

Comment:
what does this change do? the image apparently had npm before

Reply:
it has NPM now! But now it also runs in a Docker container where it won't try to install RVM and different versions of PHP and all that. This just specialized it to be a JavaScript build system.